feature/rdc-migration (#2508)
mercadopago/devsite-docs: guides/mp-point-main-apps/payment-flow-callback.en.md

# Callback Method

The callback method is a simple way to initialize the payment flow with Main Apps. It uses the new `launchPaymentFlow` method in the `PaymentFlow` class and allows for response handling through callbacks, making implementation and subsequent management easier.

> Start your integration using our SDKs to initialize the payment flow through the `PaymentFlow` class like this:

Reviewer: ```suggestion
Start your integration using our SDKs to initialize the payment flow through the `PaymentFlow` class as follows:
```

    paymentType = "paymentType",
    paymentReference = "paymentReference",
    paymentCreationDate = "paymentCreationDate",
    paymentAmount = 123,
    paymentSnDevice = "paymentSnDevice",
    paymentInstallments = "paymentInstallments",
    paymentBrandName = "paymentBrandName",
    paymentLastFourDigits = "paymentLastFourDigits",
    statusError = ""
)
```
```java
PaymentResponse paymentResponse = new PaymentResponse(
            "paymentType",
            "paymentReference",
            "paymentCreationDate",
            123,
            "paymentSnDevice",
            "paymentInstallments",
            "paymentBrandName",
            "paymentLastFourDigits",
            "statusError"
        );
```
]]]

| Field | Description |
|---|---|
|**paymentMethod**| Payment method used to perform the transaction. |
|**paymentReference**| Number that serves as a unique identifier for the transaction. |
|**paymentCreationDate**| Creation date of the transaction. |
|**Paymentamount**| Amount used to initiate the payment flow.|
|**paymentSnDevice**| Serial number of the device where the transaction was made.|
|**paymentBrandName**| Value of the franchise used to make the payment. |

> |**paymentInstallments:**| Number of installments/months that the user chose when making the payment. |

Reviewer: ```suggestion
|**paymentInstallments:**| Number of installments/months that the customer chose when making the payment. |
```
